walkthrough.ipynb

```diff
@@ -512,10 +512,11 @@
     "bart_responses = []\n",
     "\n",
     "for prompt in df['examples'][0:10]:\n",
-    "    response = ld.get_bart_ground_truth(deployment_id, prompt)\n",
+    "    response = get_bart_ground_truth(deployment_id, prompt)\n",
```

Comment:
Isn't `get_bart_ground_truth` defined in lumigator_demo? Perhaps you had a cell where you were testing it in the nb too?